util/fipstools/acvp/acvptool/subprocess/hash.go

```diff
@@ -24,6 +24,7 @@ import (
 // https://pages.nist.gov/ACVP/draft-celi-acvp-sha.html#name-test-vectors
 
 type hashTestVectorSet struct {
+	Algo   string          `json:"algorithm"`
```

Comment:
Why do we need to include a new field in the json if nothing in this go file references `Algo`? Should something be branching on this?